plugins/modules/ipahostgroup.py

```diff
@@ -328,63 +338,26 @@ def main():
                         # Set res_find to empty dict for next step
                         res_find = {}
 
-                    member_args = gen_member_args(host, hostgroup)
```

Comment:
gen_member_args is not used anymore with this change, right? It should be removed then.

Reply:
That's right. Fixed.